resources/lang/mar.ini

```diff
@@ -0,0 +1,112 @@
+command.biomeinfo.result.line = "ओळख: {%0} नाव : {%1}"
+command.biomelist.title = "बायोम यादी"
+command.biomelist.result.line = "ओळख: {%0} नाव : {%1}"
+command.brushname.set = "ब्रश नाव सेट केले आहे \"{%0}\""
+command.clearclipboard.cleared = "क्लिपबोर्ड मोकळे केले"
+command.flip.try = "{%0} द्वारा क्लिपबोर्ड फ्लिप करण्याचा प्रयत्न करीत आहे"
+command.flip.success = "क्लिपबोर्ड यशस्वीरित्या फ्लिप झाला"
+command.rotate.try = "{%0} अंशांनी क्लिपबोर्ड फिरवण्याचा प्रयत्न करीत आहे"
+command.rotate.success = "क्लिपबोर्ड यशस्वीरित्या फिरविला"
+command.history.cleared = "इतिहास पुसला"
+command.listchunks.found = "निवडीत {%0} तुकडे  सापडले"
+command.size = "निवड आकार"
+; selection
+selection.pos1.set = "जागा 1 ला X: {%0} Y: {%1} Z: {%2} वर सेट केले"
+selection.pos2.set = "जागा 2 ला X: {%0} Y: {%1} Z: {%2} वर सेट केले"
+; session
+session.undo.none = "पूर्ववत करण्यासाठी काहीही नाही"
+session.undo.left = "आपल्याकडे {%0} पूर्ववत क्रिया बाकी आहे"
+session.redo.none = "पुन्हा करण्यासारखे काही नाही"
+session.redo.left = "आपल्याकडे {%0} पुन्हा करण्याच्या  क्रिया बाकी  आहे"
+session.brush.added = "सत्रामध्ये {%0} जोडले"
+session.brush.deleted = "{%0} हटविले (UUID {%1})"
+session.brush.removed = "{%0} काढून टाकले  (UUID {%1})"
+session.language.set = "{%0} भाषा यशस्वीरित्या सेट केली"
+session.language.notfound = "{%0} भाषा आढळली नाही, म्हणून मुळ भाषा सेट करत आहोत"
+; task
+task.copy.success = "एसिंक कॉपी यशस्वी, {%0} घेतली, {%2} पैकी {%1} ब्लॉक कॉपी केले."
+task.count.success = "एसिंकचे विश्लेषण यशस्वी, {%0}  घेतले"
+task.count.result = "एकूण {%1} ब्लॉक्समध्ये {%0} ब्लॉक सापडले"
+task.fill.success = "एसिंक भरणे  यशस्वी, {%0} घेतले, {%2} पैकी {%1} ब्लॉक बदलले."
+task.replace.success = "एसिंक बदलणे यशस्वी, {%0} घेतले, {%2} पैकी {%1} ब्लॉक बदलले"
+task.revert.undo.success = "एसिंक यशस्वीरित्या पूर्ववत (undo) केले, {%0} घेतले, {%2} पैकी {%1} ब्लॉक बदलले"
+task.revert.redo.success = "एसिंक यशस्वीरित्या पुन्हा (redo) केले, {%0} घेतले, {%2} पैकी {%1} ब्लॉक बदलले"
+; flags
+flags.keepexistingblocks = "विद्यमान ब्लॉक्स ठेवा"
+flags.keepair = "हवा ठेवा"
+flags.hollow = "पोकळ"
+flags.hollowclosed = "बंद टोकांसह पोकळ"
+flags.natural = "नैसर्गिक"
+; tools
+; wand tool
+tool.wand = "कांडी"
+tool.wand.lore.1 = "निवडीची स्थिती 1 सेट करण्यासाठी ब्लॉकवर डावे क्लिक करा"
+tool.wand.lore.2 = "निवडीची स्थिती 2 सेट करण्यासाठी ब्लॉकवर उजवे क्लिक करा"
+tool.wand.lore.3 = "कार्यक्षमता बदलण्यासाठी //togglewand वापरा"
+tool.wand.disabled = "कांडीचे साधन (wand tool) निष्क्रिय केले. ते पुन्हा कार्यान्वित करण्यासाठी //togglewand वापरा"
+tool.wand.setenabled = "कांडीचे साधन आता {%0} आहे!"
+; debug tool
+tool.debug = "डीबग साधन"
+tool.debug.lore.1 = "माहिती मिळविण्यासाठी ब्लॉक वर डावे क्लिक करा"
+tool.debug.lore.2 = "ब्लॉकचे नाव आणि नुकसान मूल्ये यासारख्या"
+tool.debug.lore.3 = "कार्यक्षमता बदलण्यासाठी //toggledebug वापरा"
+tool.debug.disabled = "डीबग साधन निष्क्रिय केले आहे. ते पुन्हा सक्रिय करण्यासाठी //toggledebug वापरा"
+tool.debug.setenabled = "{%0} आता डीबग साधन आहे!"
```

Comment:
Maybe the position of {%0} (which will be "enabled"/"disabled") should be similar to the tool.wand.setenabled entry

Reply:
Yes, you are right!
For the first pass I went with literal meaning. Above comment made me understand the the correlation between `enabled/disabled` and `tool.debug.setenabled `. 
Fixed it now.